tasty-query/shared/src/main/scala/tastyquery/Symbols.scala

```diff
@@ -135,6 +134,11 @@ object Symbols {
         assert(false, s"cannot access owner, ${this.name} is local or not declared within any scope")
     }
 
+    /** What is the closest enclosing package of this symbol? Returns this if this is a package */
```

Comment:
```suggestion
    /** The closest enclosing package of this symbol.
      * Returns this if this is a package.
      */
```
Don't frame Scaladoc comments as questions.